### PR TITLE
Improve list display for deliveries

### PR DESCRIPTION
**Pull request by Atala**
Use colors to show clearly linked tasks (a.k.a deliveries for the moment)

capture d ecran 2018-04-20 a 18 24 44

capture d ecran 2018-04-20 a 18 24 19

### CODE REVIEW

File: src/AppBundle/Entity/Delivery.php [hunk 1]
```diff
@@ -45,6 +45,8 @@ class Delivery extends TaskCollection implements TaskCollectionInterface
     const VEHICLE_BIKE = 'bike';
     const VEHICLE_CARGO_BIKE = 'cargo_bike';
 
+    const COLORS_LIST = ['#213ab2', '#b2213a', '#5221b2', '#93c63f', '#b22182', '#3ab221', '#b25221', '#2182b2', '#3ab221', '#9c21b2', '#c63f4f', '#b2217f', '#82b221', '#5421b2', '#3f93c6', '#21b252', '#c6733f'];
```
Comment (alexsegura) [inline src/AppBundle/Entity/Delivery.php]: This is too much coupling between the view & the model. 
This should be done in Twig / JavaScript side.

File: src/AppBundle/Resources/config/doctrine/TaskCollectionItem.orm.yml [hunk 2]
```diff
@@ -27,7 +27,7 @@ AppBundle\Entity\TaskCollectionItem:
         task:
             targetEntity: AppBundle\Entity\Task
             cascade:
-                - persist
+                - all
```
Comment (alexsegura) [inline src/AppBundle/Resources/config/doctrine/TaskCollectionItem.orm.yml]: This doesn't seem related to this PR.

**Comment by alexsegura** [comment]
The UI is cool ⭐️

File: src/AppBundle/Entity/Delivery.php (hunk 1 above)
Comment (Atala) [reply src/AppBundle/Entity/Delivery.php]: Advantage to having it here is that it is easy to serialize `color` in the API (so we have exactly same colors everywhere).

Maybe having a service that get the color is a better solution?
